.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,39 @@
-Issue:
+<!-- Thank you for contributing to Storybook! If your PR is related to an issue, provide the number(s) here, e.g. #1000, #1001 -->
+Issue: #
+
 
 ## What I did
 
+<!-- Briefly describe what your PR does -->
+
+
 ## How to test
 
-- [ ] Is this testable with Jest or Chromatic screenshots?
-- [ ] Does this need a new example in the kitchen sink apps?
-- [ ] Does this need an update to the documentation?
+<!-- Please include the steps to test your changes here. For example:
 
-If your answer is yes to any of these, please make sure to include it in your PR.
+1. Run a sandbox for template, e.g. `yarn task --task sandbox --start-from auto --template react-vite/default-ts`
+2. Open Storybook in your browser
+3. Access X story
 
-<!--
+-->
 
-Everybody: Please submit all PRs to the `next` branch unless they are specific to the current release. Storybook maintainers cherry-pick bug and documentation fixes into the `master` branch as part of the release process, so you shouldn't need to worry about this. For additional guidance: https://storybook.js.org/docs/react/contribute/how-to-contribute
 
-Maintainers: Please tag your pull request with at least one of the following:
-`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
+## Checklist
+<!-- Please check (put an "X" inside the "[ ]") the applicable items below to make sure your PR is ready to be reviewed. -->
+
+- [ ] Make sure your changes are tested (stories and/or unit, integration, or end-to-end tests)
+- [ ] Make sure to add/update documentation regarding your changes
+- [ ] If you are deprecating/removing a feature, make sure to update
+      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)
+
+#### Maintainers
+- [ ] If this PR should be tested against many or all sandboxes,
+      make sure to add the `ci:merged` or `ci:daily` GH label to it.
+- [ ] Make sure this PR contains the correct GH labels
+<!-- with at least one of the following `["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]` -->
```

Comment:
1. I still struggle to figure out which of these labels are appropriate for my PRs. I miss a sentence or two about each of them, and I guess this is a pretty good place to have that.?

2. I might be wrong, but I think CI fails if there are multiple of these labels on. If that is true we should rephrase it.